From b7982df5e19d28087122425a091317b29d24dce6 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 06 Feb 2025 14:18:51 +0200
Subject: [PATCH] [SW-0] g3: add hl325l to hl_sensors_internal.conf

HL325L's sensors doesn't support be the development sensors' alternative
file.

Change-Id: I2a554f6e2e404a11bf6109f758f99f44b080f5bb
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/sensors/hl_sensors_internal.conf b/package/sensors/hl_sensors_internal.conf
index 5da5ca0..f165f40 100644
--- a/package/sensors/hl_sensors_internal.conf
+++ b/package/sensors/hl_sensors_internal.conf
@@ -202,7 +202,7 @@
 # fix resolution of power to be displayed in watts
 compute power1 (@ * 1000), (@ * 1000)
 
-chip "HL325-pci-*"
+chip "HL325-pci-*" "HL325L-pci-*"
 
 label temp1 "On Chip 0"
 label temp2 "On Chip 0 CH0"
